2021-09-12 20:46:31,453 - no description
2021-09-12 20:46:31,453 - dataset:ml-1m lr1:0.001 lr2:0.0003 kl:0.05 contrast:0.1 batch:1 model_func:fc_att
2021-09-12 20:46:31,453 - Using CUDA:0
2021-09-12 20:46:32,534 - Building net...
2021-09-12 20:46:32,547 - Net build finished.
2021-09-12 20:46:32,547 - User datasets loaded and saved.
2021-09-12 20:46:32,549 - Start training...
2021-09-12 20:46:32,578 - Traceback (most recent call last):
  File "trainMe0912.py", line 306, in <module>
    train()
  File "trainMe0912.py", line 214, in train
    pred, x_real, z_inferred, out_embed = net(batchx)
  File "/root/anaconda3/envs/llpytorch/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/lab30/lylee/ACVAE-PyTorch-main/Federated_train1+(gru+gruPooling)/modelMe0912.py", line 301, in forward
    rnn_out = rnn_out + x
RuntimeError: The size of tensor a (100) must match the size of tensor b (128) at non-singleton dimension 2

2021-09-12 20:46:32,578 - Error.
